extension/src/NABfunctions.ts

```diff
@@ -994,3 +1059,9 @@ export async function importDtsTranslations(): Promise<void> {
 
   console.log("Done: importDtsTranslations");
 }
+
+interface IExportOptions {
+  columns: string[];
```

Comment:
Now that the columns are being used i several places, wouldn't it make sense to have an Enum instead?